README.md

```diff
@@ -20,56 +20,33 @@ Prerequisites
 -------------
 
 
-### Linux
+### Linux and Mac OS 
 
 For Ginkgo core library:
 
-*   _cmake 3.1+_
-*   C++11 compliant compiler, one of:
-    *   _gcc 5.4.0+_
-    *   _clang 3.3+_ (__TODO__: verify, works with 5.0)
-
-The Ginkgo CUDA module has the following __additional__ requirements:
-
-*   _cmake 3.10+_
-*   _CUDA 7.0+_ (__TODO__: verify, works with 8.0)
-*   Any host compiler restrictions your version of CUDA may impose also apply
-    here. For the newest CUDA version, this information can be found in the
-    [CUDA installation guide](http://docs.nvidia.com/cuda/cuda-installation-guide-linux/index.html#system-requirements)
-
-In addition, if you want to contribute code to Ginkgo, you will also need the
-following:
-
-*   _clang-format 5.0.1+_ (ships as part of _clang_)
-
-
-### Mac OS
-
-For Ginkgo core library:
-
-*   _cmake 3.1+_
+*   _cmake 3.8+_
 *   C++11 compliant compiler, one of:
-    *   _gcc 5.4.0+_ (__TODO__: verify)
-    *   _clang 3.3+_ (__TODO__: verify)
+    *   _gcc 5.5.0+_
+    *   _clang 3.9+_
     *   _Apple LLVM 8.0+_ (__TODO__: verify)
 
 The Ginkgo CUDA module has the following __additional__ requirements:
 
-*   _cmake 3.8+_
-*   _CUDA 7.0+_ (__TODO__: verify)
+*   _CUDA 9.0+_
 *   Any host compiler restrictions your version of CUDA may impose also apply
     here. For the newest CUDA version, this information can be found in the
-    [CUDA installation guide](http://docs.nvidia.com/cuda/cuda-installation-guide-mac-os-x/index.html)
+    [CUDA installation guide for Linux](https://docs.nvidia.com/cuda/cuda-installation-guide-linux/index.html#system-requirements)
```

Comment:
```suggestion
    [CUDA installation guide for Linux](https://docs.nvidia.com/cuda/cuda-installation-guide-linux/index.html)
```
I would just drop the section link to `system-requirements` (even though if it was in before), so you can see the page from the beginning (and not skip the small introduction).